src/main/resources/swagger/api-docs.yaml

```diff
@@ -221,6 +221,10 @@ paths:
           description: Snapshot ID of configuration.
           required: false
           type: number
+        - name: snapshotComment
```

Comment:
This doesn't appear to be `required`, similar to the Agora PR. Can this be annotated similarly?

Reply:
My bad, this is required - ignore comment. Pretty sure this also constitutes a breaking change to the API - we will need to handle that with a release email.

Reply:
It should not be `required`, snapshot comments are designed to be very optional. Will fix & check consistency across PRs.

Reply:
👍  - Yes, as I dig more into the agora PR, it is becoming more clear, thanks!

Reply:
The default value of `required` is `false`, so this was correct, but I'm going to make it explicit.